packages/monaco/src/browser/monaco-keybinding.ts

```diff
@@ -79,7 +79,7 @@ export class MonacoKeybindingContribution implements KeybindingContribution {
             first: Key.getKey(monaco2BrowserKeyCode(keyCode & 255)),
             modifiers: []
         };
-        if (keybinding.ctrlKey) {
+        if (keybinding.metaKey) {
```

Comment:
I think with this change, we never add Ctrl to the sequence in Windows... I am verifying it anyways

Reply:
We rely on the mapping logic from monaco:
https://github.com/Microsoft/vscode/blob/master/src/vs/base/common/keyCodes.ts#L428

Reply:
The logic should be something like this 
 - `IF OS === macOS AND isMeta AND isCtrl` then it is a `CtrlCmd`
 - `IF OS === macOS AND !isMega AND isCtrl` the it is a `MacCtrl` (this should not do anything on Windows)
 - `IF OS !== macOS AND isCtrl` then it is a `CtrlCmd`

Reply:
I see. So it is already handled by monaco. Thanks for the clarification.